marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/lib.sjs

```diff
@@ -271,6 +271,10 @@ function validatePropertyMapping(targetEntityType, propertyName, sourcedFrom) {
 }
 
 function versionIsCompatibleWithES(version = xdmp.version()) {
+  // Turns out 9.0-10.2 is fine, but 9.0-10 is not
```

Comment:
9.0-10.1 isn't either, as n fyi. Nor is 10.01

Reply:
I didn't see a 9.0-10.1 in mlregistry.marklogic.com

Reply:
It's a released version. Trust me on that. Min versions for this stuff should be 10.02 and 9.10-2 or later.